### PR TITLE
[WIP] Migrate i18n check scripts from activist/frontend repo to this checks......

### PR DESCRIPTION
**Pull request by OmarAI2003**
… repo without modifications.

### Contributor checklist

- [x] This pull request is on a [separate branch](https://docs.github.com/en/get-started/quickstart/github-flow) and not the main branch

---

### Description

This is an initial commit to migrate the i18n check files from the activist/frontend repo to this repository. Let me know the next specific steps to proceed!

### Related issue

- #1

### CODE REVIEW

**Comment by andrewtavis** [comment]
So far so good, @OmarAI2003! Do you want to start bringing over versions of the main package files? We could use Scribe-Data as an example here, which would allow you to practice building a Python package from scratch?

If so, let's bring over the following files and edit them to work over here:

- MANIFEST.in
- pyproject.toml
- setup.cfg
- setup.py

If we want to do readthedocs for docs here - likely an easy solution - then you can also bring over:
- .readthedocs.yaml

And would be nice to also do pre-commit here, so we'd do:
- .pre-commit-config.yaml

Feel free to copy over the Scribe-Data files and change things as you see fit. I'll let you know in a review if all's looking ok 😊

**Comment by OmarAI2003** [comment]
I've added the initial configuration files as discussed.
@andrewtavis  Let me know if any adjustments are needed or what the next steps should be.

**Comment by andrewtavis** [comment]
I think we'll be good to merge this and move on to #2 once the comments above are addressed, @OmarAI2003! 🚀

**Review by andrewtavis — APPROVED** [review]
All's looking great here, @OmarAI2003! Thanks so much for the initial migration work  here :) Looking forward to removing the code from activist and installing this 😊

File: src/i18n_check/checks/i18n_check_key_identifiers.py
Comment (andrewtavis) [reply src/i18n_check/checks/i18n_check_key_identifiers.py:98]: Ya all of this would be great to include! Thanks, @OmarAI2003! Looking forward to the PR 😊

**Pull request merged**
